Kudu.Core/Jobs/TriggeredJobsManager.cs

```diff
@@ -31,7 +32,18 @@ public TriggeredJobsManager(ITraceFactory traceFactory, IEnvironment environment
 
         public override IEnumerable<TriggeredJob> ListJobs()
         {
-            return ListJobsInternal();
+            var cache = TriggeredJobCache;
+            if (cache == null)
+            {
+                cache = ListJobsInternal();
+                TriggeredJobCache = cache;
+            }
+            return cache;
+        }
```

Comment:
This method looks identical to the Continuous one. Shouldn't it just be in the base as an `IEnumerable<TJob>` to avoid dup?